### PR TITLE
Support template ID as an issue creation input

### PR DESCRIPTION
**Pull request by leelasn**
Now that our API supports `templateId` as an input during issue creation, this is relatively straightforward. I pull the title from the template if it has one, but you have to select team yourself since Zapier doesn't let me use a function to set the value of a dynamic dropdown.

### CODE REVIEW

File: src/creates/createIssue.ts [hunk 1]
```diff
@@ -197,11 +210,30 @@ export const createIssue = {
...
+            query ZapierTemplate($templateId: String!) {
+              template(id: $templateId) {
+                templateData
+              }
+            }`;
+          const response = await fetchFromLinear(z, bundle, query, { templateId: bundle.inputData.templateId });
+
+          const template = response.json.data.template;
+          const templateData = JSON.parse(template.templateData);
+          if (templateData.title) {
```
Comment (tommoor) [inline src/creates/createIssue.ts:232]: To confirm – the template issue title would be used even without this logic, this is just a nicety to display it in Zapier? Or is there more to it.

Comment (leelasn) [reply src/creates/createIssue.ts:232]: It’s also because we mark title as a required field here

Comment (tommoor) [reply src/creates/createIssue.ts:232]: I suppose the downside is if the template title changes inside of Linear the Zapier will be stuck with the old title.

Anyway to make title optional based on existence of another param?

Comment (leelasn) [reply src/creates/createIssue.ts:232]: Let me see

Comment (leelasn) [reply src/creates/createIssue.ts:232]: Got it to work

Comment (tommoor) [reply src/creates/createIssue.ts:232]: lovely

**Review by tommoor — APPROVED** [review]

**Pull request merged**